donate.html

```diff
@@ -2049,91 +2043,93 @@ <h4>Shipping Address</h4>
         },
-        submitOnEmpty: function () {
-          return $('#ak-pay-by-paypal').val() == 1;
-        },
-        submit: form.querySelector(".ak-submit-button")
-      },
-      toRemove = ["#ak-card_num", "#ak-card_code", "#ak-exp_date_month",
-        "#ak-exp_date_year", "#ak-card_num-required",
-        "#ak-exp_date_month-required",
-        "#ak-exp_date_year-required", "#ak-card_code-required"];
-    toRemove.forEach(function (el) {
-      $(el).remove();
-    });
+        toRemove = ["#ak-card_num", "#ak-card_code", "#ak-exp_date_month",
+          "#ak-exp_date_year", "#ak-card_num-required",
+          "#ak-exp_date_month-required",
+          "#ak-exp_date_year-required", "#ak-card_code-required"];
+      toRemove.forEach(function (el) {
+        $(el).remove();
+      });
 
-    // Fix to stop donation errors gettin redirected from the error page
-    actionkit.utils.appendHiddenInput('referer_from_url', '1');
-    actionkit.donations.handleError = function (err) {
-      if (window.console) {
-        window.console.log(err);
-      }
-      if (!actionkit.errors) {
-        actionkit.errors = {};
-      }
-      switch (err.code) {
-        case 'HOSTED_FIELDS_FIELDS_EMPTY':
-          actionkit.errors['card_num:invalid'] = "Credit card fields are empty.";
-          break;
-        case 'HOSTED_FIELDS_FIELDS_INVALID':
-          err.details.invalidFieldKeys.forEach(function (key) {
-            var map = {
-              'number': 'card_num',
-              'cvv': 'card_code',
-              'expirationDate': 'card_exp'
-            };
-            actionkit.errors[map[key] + ':invalid'] = "This field is invalid.";;
-          });
-          break;
-        case 'HOSTED_FIELDS_FAILED_TOKENIZATION':
-          actionkit.errors['card_num:invalid'] = "Please check that your credit card is valid.";
-          break;
-        case 'HOSTED_FIELDS_TOKENIZATION_NETWORK_ERROR':
-          actionkit.errors['card_num:invalid'] = "Network error.";
-          break;
-        default:
-          actionkit.errors['card_num:invalid'] = "An error occurred: " + err.message;
-      }
-      actionkit.forms.onValidationErrors(actionkit.errors);
-    };
-    actionkit.donations.initClient('{{ pp.client_token }}', options);
-    // read by 3-step validation
-    actionkit.donations.skip_cc_validation = true;
-    actionkit.donations.vzero = true;
-    $(".hosted-field.braintree-hosted-fields-invalid").prev().css("background-color", "#fff0d4");
-    {% endif %}
+      // Fix to stop donation errors gettin redirected from the error page
+      actionkit.utils.appendHiddenInput('referer_from_url', '1');
+      actionkit.donations.handleError = function (err) {
+        if (window.console) {
+          window.console.log(err);
+        }
+        if (!actionkit.errors) {
+          actionkit.errors = {};
+        }
+        switch (err.code) {
+          case 'HOSTED_FIELDS_FIELDS_EMPTY':

```

Comment:
This is a nit pick but for variables like these, I often put them in a map /dictionary to ensure I avoid small mishaps e.g with typos. Not an important change so feel free to ignore this.

Reply:
Definitely agree - I think this is another change we can make in future PRs!